muxink/src/codec/bytesrepr.rs

```diff
@@ -0,0 +1,172 @@
+//! Bytesrepr encoding/decoding
```

Comment:
Can we add a little more info on this module, potential pitfalls and such, if there are any?

... and a blank line? :)

Reply:
Comments updated in 0448111c27126081d80f60cff3b8a7bb9b456483. The comment is rather dull (that's why I decided to not include it in the first place).

Blank line has been added as well.